rust/core-lib/src/plugins/rpc.rs

```diff
@@ -75,6 +75,10 @@ pub struct PluginUpdate {
     /// The total number of lines in the document after applying this delta.
     pub new_line_count: usize,
     pub rev: u64,
+    /// The undo_group associated with this update. Each PluginEdit sent by plugin is associated
```

Comment:
I would reword the second sentence a little, to say:

"The plugin may pass this value back to core when making an edit, to associate the plugin's edit with this undo group." (then continue with your sentence that starts, "Core uses ...")